htlcswitch/link.go

```diff
@@ -982,10 +982,15 @@ func (l *channelLink) handleDownStreamPkt(pkt *htlcPacket, isReProcess bool) {
 					reason       lnwire.OpaqueReason
 				)
 
-				failure := lnwire.NewTemporaryChannelFailure(nil)
+				update, _ := l.cfg.FetchLastChannelUpdate(
+					l.shortChanID,
```

Comment:
`shortChanID` is protected by a mutex, which has not been acquired here. One could argue that the value should not change after the link has been started, though might be best to use `l.ShortChanID()`